recipes/renode-cli/recipe.yaml

```
@@ -0,0 +1,74 @@
+context:
+  name: renode-cli
+  version: "1.15.3"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/renode/renode/releases/download/v${{ version }}/renode_${{ version }}_source.tar.xz
+    sha256: 40c50afe8db86a9c63f4a86b93116a6babd43bf241cfaac47b2c0d2cbc2b15a8
+    patches:
+      - patches/update-cores.template_NET.csproj.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - dotnet
+    - nuget-license
+    - if: unix
+      then:
+        - sed
+        - bash
+      else:
+        - m2-bash
+        - m2-coreutils
+        - m2-findutils
+        - m2-sed
+  host:
+    - dotnet
+    - renode-cores ==${{ version }}
+  run:
+    - dotnet
+
+tests:
+  - package_contents:
+      bin:
+        - renode
+  - script:
+      - renode --help
+      - LOCAL_TEST_PREFIX=./test-bundle test-bundle/bin/renode-test test-bundle/share/renode-cli/tests/unit-tests
+    requirements:
+      run:
+        - robotframework 6.1
+        - psutil 5.9.3
+        - python
+        - pyyaml 6.0
```

```suggestion
        - robotframework 6.1
        - psutil 5.9.3
        - python
        - pyyaml 6.0
```
How strict should these bounds be?  If possible it would be better to set ranges like `psutil >=5.9,<5.10`, and only if the latest version doesn't work.

---

Actually, the original recipe does not have requirements, but for robotframework. I may have had issues and tuned to the minimal revision., we can leave them open ended